examples/nextjs-app-router/components/Form.tsx

```diff
@@ -17,15 +17,29 @@ import {
 	validateForm,
 } from "simple-stack-form/module";
 import { twMerge } from "tailwind-merge";
+import { useFormState as useForStateBase } from "react-dom";
+
+type ValidateFormResult = Awaited<ReturnType<typeof validateForm>> | null;
+
+export function useFormState(
+	action: (formData: FormData) => Promise<ValidateFormResult>,
+) {
+	return useForStateBase(
```

Comment:
Clever! I'm curious if people will like this wrapper. I'll use it for demos personally